toasty/collection.py

```diff
@@ -54,6 +62,34 @@ def images(self):
         """
         raise NotImplementedError()
 
+    def is_multi_tan(self):
```

Comment:
I think this function is probably best thought of as an internal interface for now, so I think the name should be underscored `_is_multi_tan()` to mark it as private (at least, as private as things get in Python).